homeassistant/helpers/entity_platform.py

```diff
@@ -623,11 +623,16 @@ async def async_add_entities(
 
         self._async_unsub_polling = async_track_time_interval(
             self.hass,
-            self._update_entity_states,
+            self._async_update_entity_states,
             self.scan_interval,
             name=f"EntityPlatform poll {self.domain}.{self.platform_name}",
         )
 
+    @callback
+    def _async_update_entity_states(self, now: datetime) -> None:
```

Comment:
The name is a bit confusing. `async_update_entity_states` and `_update_entity_states`. Maybe call the first one something like `async_handle_interval_callback`?